sdk/storage/storage-queue/src/utils/tracing.ts

```diff
@@ -14,7 +14,7 @@ export function createSpan(
   tracingOptions: OperationTracingOptions = {}
 ): { span: Span; spanOptions: SpanOptions } {
   const tracer = getTracer();
-  const spanOptions: SpanOptions = {
+  const spanOptions: OTSpanOptions = {
```

Comment:
Why even bother typing this?
Can't we make do with `const spanOptions = { ...}`

Reply:
I think this was done to enforce some strictness in this helper (both to get IntelliSense on authoring and to guard against missed renames on OT changes), but I am fine with removing these for now. 

Will push an update with this change soon.

Reply:
OK the problem is TS at build time will complain:

```
src/utils/tracing.ts:31:24 - error TS2339: Property 'attributes' does not exist on type '{ kind: SpanKind; } | { kind: SpanKind; parent?: SpanContext | null | undefined; attributes?: { [key: string]: unknown; } | undefined; }'.
  Property 'attributes' does not exist on type '{ kind: SpanKind; }'.
```

To force it to accept broadening the type to what is splatted onto it, we have to explicitly give it the type

Reply:
fine!